picky-krb/src/crypto/nfold.rs

```diff
@@ -0,0 +1,166 @@
+#[inline]
```

Comment:
You don’t need the #[inline] attribute when the function is used inside the same crate (this is only useful for inlining across crates). There is `#[inline(always)]` to override the inlining heuristic, but I advise against.

Reply:
Done